spec/controllers/category_controller_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+RSpec.describe CategoriesController, type: :controller do
+  let(:categories) { FactoryGirl.create(:category) }
+  describe "GET index" do
+    it "it renders index action" do
```

Comment:
**Nitpick - Code Quality**
You don't need to append `it` to the description.
You can have `it "renders based on the index action" do`

Please fix all instances of this issue.